fluent/ExtrinsicRune.ts

```diff
@@ -59,13 +59,29 @@ export class ExtrinsicRune<out C extends Chain, out U> extends PatternRune<Chain
       .fn(concat)
       .call(extrinsic, extrinsic.access("length").map((n) => $.u32.encode(n)))
       .map(hex.encodePrefixed)
-    const data = this.chain.connection
+    // The TransactionPaymentApi_query_info endpoint returns a heavily-padded hex str.
+    // If we don't strip the excess before passing to other calls, we get a runtime panic.
+    return this.chain.connection
       .call("state_call", "TransactionPaymentApi_query_info", arg)
```

Comment:
(`TransactionPaymentApi_query_info` doesn't directly return a weight, it returns a `RuntimeDispatchInfo`)[https://github.com/paritytech/substrate/blob/master/frame/transaction-payment/rpc/runtime-api/src/lib.rs#L32]

So it's not a "heavily-padded hex str", but rather has other data. It should be decoded with a `$.tuple($weight, $dispatchClass, $balance)`.

The initial `$weight` won't be consistently 6 bytes, as it is `$.tuple($.compact($.u128), $.compact($.u128))`, so the size varies based on the values.